db/migrate/20160510135800_create_items.rb

```diff
@@ -0,0 +1,13 @@
+class CreateItems < ActiveRecord::Migration
+  def change
+    create_table :items do |t|
+      t.string   :name, null: false, limit: 50
+      t.text     :description, null: false, limit: 250
```

Comment:
The string maximum limit is 255. Why do you need a text field that you are trying to limit to 250? The limit doesn't seem to apply if you look at the generated schema.